### PR TITLE
chore: Add win 210218 release notes

### PR DESCRIPTION
**Pull request by mainred**

### CODE REVIEW

**Review by AbelHu — APPROVED** [review]
I think that we should merge this after merging the change in RP. Otherwise, next AKS release notes may include this release notes for the new VHD but the AKS release does not have this VHD.

**Comment by mainred** [comment]
Close this PR since we did not enable this VHD in AKS

**Pull request closed without merging**
